npm/private/utils.bzl

```diff
@@ -106,11 +109,17 @@ def _convert_v6_packages(packages):
     # Convert pnpm lockfile v6 importers to a rules_js compatible format.
     result = {}
     for package, package_info in packages.items():
-        new_deps = {}
-        deps = package_info.get("dependencies", {})
-        for dep_name, dep_version in deps.items():
-            new_deps[dep_name] = _convert_pnpm_v6_package_name(dep_version)
-        package_info["dependencies"] = new_deps
+        # dependencies
+        dependencies = {}
+        for dep_name, dep_version in package_info.get("dependencies", {}).items():
+            dependencies[dep_name] = _convert_pnpm_v6_package_name(dep_version)
+        package_info["dependencies"] = dependencies
+
+        # optionalDependencies
```

Comment:
What about dev, peer etc? Are those not in the lockfile? I see `devDependencies` in the method above...

Reply:
They are not in the packages. Resolved dev & peer deps go into dependencies.